external/netfx/Configurations.props

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <!-- Include netcoreapp since we need to generate facades between desktop and netcoreapp -->
+    <BuildConfigurations>
+      net45;
+      net451;
+      net46;
+      net461;
+      net462;
+      net463;
```

Comment:
>net463 [](start = 6, length = 6)

There is no .NET Framework 4.6.3. Should we have net471?

Reply:
It is what CoreFx is using...
https://github.com/dotnet/corefx/blob/release/2.0.0/external/netfx/Configurations.props

Reply:
I believe net461 is the only one you guys need.